ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -176,7 +176,7 @@ def update_charon_with_local_jobs_status(config=None, config_file_path=None):
 
 @with_ngi_config
 def recurse_status_for_sample(project_obj, set_status, update_done=False,
-                              config=None, config_file_path=None):
+                              extra_args={},config=None, config_file_path=None):
```

Comment:
You gotta change this to be `extra_args=None` in the function definition, and then do a check in the code `if not extra_args: extra_args = {}`